.gitignore

```diff
@@ -0,0 +1,9 @@
+# Django #
+*.log
+local_settings.py
+db.sqlite3
+db.sqlite3-journal
+*.pot
+*.pyc
+__pycache__/
+media
```

Comment:
Let's add a new line at the end of every file that shows "No newline at EOF" symbol.

Reply:
okay